packages/react-core/src/components/Label/examples/Label.md

```diff
@@ -7,15 +7,251 @@ propComponents: ['Label']
+    <br />
+    <Label color="red">Red</Label>{' '}
+    <Label color="red" icon={<InfoCircleIcon />}>
+      Red icon
+    </Label>{' '}
+    <Label color="red" onClose={Function.prototype}>
+      Red removeable
+    </Label>{' '}
+    <Label color="red" icon={<InfoCircleIcon />} onClose={Function.prototype}>
+      Red icon removeable
+    </Label>{' '}
+    <Label color="red" href="#">
+      Red link
+    </Label>{' '}
+    <Label color="red" href="#" onClose={Function.prototype}>
+      Red link removeable
+    </Label>
+    <br />
+    <br />
+    <Label color="purple">Purple</Label> <Label color="purple" icon={<InfoCircleIcon />}>
+      Purple icon
+    </Label> <Label color="purple" onClose={Function.prototype}>
+      Purple removeable
+    </Label> <Label color="purple" icon={<InfoCircleIcon />} onClose={Function.prototype}>
+      Purple icon removeable
+    </Label> <Label color="purple" href="#">
+      Purple link
+    </Label> <Label color="purple" href="#" onClose={Function.prototype}>
+      Purple link removeable
+    </Label>
+    <br />
+    <br />
+    <Label color="cyan">Cyan</Label>{' '}
+    <Label color="cyan" icon={<InfoCircleIcon />}>
+      Cyan icon
+    </Label>{' '}
+    <Label color="cyan" onClose={Function.prototype}>
+      Cyan removeable
+    </Label>{' '}
+    <Label color="cyan" icon={<InfoCircleIcon />} onClose={Function.prototype}>
+      Cyan icon removeable
+    </Label>{' '}
+    <Label color="cyan" href="#">
+      Cyan link
+    </Label>{' '}
+    <Label color="cyan" href="#" onClose={Function.prototype}>
+      Cyan link removeable
+    </Label>
+  </React.Fragment>
+);
+```
+
+```js title=Outline
 import React from 'react';
 import { Label } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
 
-SimpleLabel = () => (
+FilledLabels = () => (
   <React.Fragment>
-    <Label>Default Label</Label> <Label isCompact>Compact Label</Label>
+    <Label variant="outline">Grey</Label>{' '}
+    <Label variant="outline" icon={<InfoCircleIcon />}>
+      Grey icon
+    </Label>{' '}
+    <Label variant="outline" onClose={Function.prototype}>
+      Grey removeable
+    </Label>{' '}
+    <Label variant="outline" icon={<InfoCircleIcon />} onClose={Function.prototype}>
+      Grey icon removeable
+    </Label>{' '}
+    <Label variant="outline" href="#">
+      Grey link
+    </Label>{' '}
+    <Label variant="outline" href="#" onClose={Function.prototype}>
+      Grey link removeable
+    </Label>
+    <br />
+    <br />
+    <Label variant="outline" color="blue">
+      Blue
+    </Label> <Label variant="outline" color="blue" icon={<InfoCircleIcon />}>
+      Blue icon
+    </Label> <Label variant="outline" color="blue" onClose={Function.prototype}>
+      Blue removeable
+    </Label> <Label variant="outline" color="blue" icon={<InfoCircleIcon />} onClose={Function.prototype}>
+      Blue icon removeable
+    </Label> <Label variant="outline" color="blue" href="#">
```

Comment:
Would it make sense to set the `href`s to `#filled` or `#outline` for the link labels in both examples? At the moment when I click an outlined label with a link, it takes me back up to the top of the page.